modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/related/models/test/ObjectRelatedModelsProviderTest.java

```diff
@@ -454,6 +473,178 @@ private void _setUser(User user) {
 		PrincipalThreadLocal.setName(user.getUserId());
 	}
 
+	private void _testSystemObjectEntry1toMObjectRelatedModels()
+		throws Exception {
+
+		long[] userIds = _addUsers(3);
+
+		ObjectDefinition systemObjectDefinition = null;
+
+		for (ObjectDefinition objectDefinition :
+				_objectDefinitionLocalService.getSystemObjectDefinitions()) {
+
+			if (StringUtil.equals(objectDefinition.getName(), "User")) {
+				systemObjectDefinition = objectDefinition;
+			}
+		}
+
```

Comment:
@p-albuquerque can you do an assert here checking if the systemObjectDefinition is not null?